expose_ocp_api.sh

```diff
@@ -22,6 +22,6 @@ if ! openstack floating ip list | grep ${LB_FLOATING_IP} ; then
 fi
 openstack floating ip set --port lb-port ${LB_FLOATING_IP}
 
-if ! grep -q ${LB_FLOATING_IP} /etc/resolv.conf ; then
-    (echo "nameserver ${LB_FLOATING_IP}" && cat /etc/resolv.conf) | sudo tee /etc/resolv.conf
+if ! grep -q ${LB_FLOATING_IP} /etc/hosts ; then
+    (echo "${LB_FLOATING_IP} ostest-api.shiftstack.com" && cat /etc/hosts) | sudo tee /etc/hosts
```

Comment:
Hmm not sure about this - don't we need both, e.g a way to reference the ostest-api node by name, and then a way to reference the DNS running on it, e.g to resolve names for apps deployed on the resulting OpenShift?  cc @flaper87 @russellb

Reply:
Yeah, I guess we'll need both.  We can't actually run apps yet though, so this is fine for now.  We'll probably want to re-add /etc/resolv.conf later.